composer.json

```diff
@@ -28,13 +28,13 @@
   },
   "require": {
     "php": ">=5.4",
-    "composer-plugin-api": "^1.0 || ^2.0",
+    "composer-plugin-api": "^2.2",
```

Comment:
For the record - as I had to double-check this myself:

**The plugin API _does_ follow the same _major_ version as Composer has, but does NOT necessarily align to the same _minors_.**

The current version of the `composer-plugin-api` is `2.6.0` while the Composer package itself is at `2.8.4`. See:
https://github.com/composer/composer/blob/f5e7a8390d700d09c87ae2ed79d5aea1e4153cd3/src/Composer/Plugin/PluginInterface.php#L35

However, in this specific case, the versions happen to line up for the `2.2.0` version of Composer:
https://github.com/composer/composer/blob/e174a4c4324f50a6f2de472aa1055c24a2fe2b2a/src/Composer/Plugin/PluginInterface.php#L35